Sources/StringsLintFramework/Rules/Lint/JSONCommentRule.swift

```diff
@@ -199,7 +215,7 @@ extension JSONCommentRule {
                 return "missingDescription"
             case .emptyDescription:
                 return "emptyDescription"
-            case .containsInvalidPlaceholders(let invalidPlaceholders):
+            case .containsInvalidPlaceholders(let _):
```

Comment:
nit: you can just remove the whole brackets
```suggestion
            case .containsInvalidPlaceholders:
```